test/unit/sql/index.test.js

```diff
@@ -147,4 +147,14 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
       });
     }
   });
+
+  suite('removeIndex', function () {
+    test('naming', function () {
+      expectsql(sql.removeIndexQuery('table', ['column1', 'column2'], {}, 'table'), {
+        mysql: 'DROP INDEX `table_column1_column2` ON `table`',
+        mssql: 'DROP INDEX table_column1_column2 ON [table]',
```

Comment:
It should likely be `DROP INDEX [table_column1_column2] ON [table]` to apply the same fix as has been done to other dialects.